portfolio/src/main/webapp/index.html

```diff
@@ -2,14 +2,28 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Isabel's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <h1>Isabel's Portfolio</h1>
+      <p>Hi! My name is Isabel and I am excited to be a STEP intern at Google this summer.
+         On this page you can learn a little bit more about me.
+      <h2>Background</h2>
+      <p>I was born and raised in New Jersey and am currently working from home in NJ. Jersey gets a 
+         pretty bad rap sometimes--people have told me it is the "Armpit of America" or "the ugly 
+         stepchild of New York"-- but I love it here.  We have everything you could want in a state:
```

Comment:
Nit: As an em dash enthusiast, I would advise using &mdash; or — instead of --. I prefer the AP style of having a space on either side, but being the most important part of the chosen style is being consistent.

```suggestion
         pretty bad rap sometimes &mdash; people have told me it is the "Armpit of America" or "the ugly 
         stepchild of New York" &mdash; but I love it here.  We have everything you could want in a state:
```